Sources/xcodeproj/Scheme/XCScheme+ArchiveAction.swift

```diff
@@ -47,13 +47,15 @@ extension XCScheme {
         }
 
         // MARK: - Equatable
-
-        override func isEqual(to: Any?) -> Bool {
-            guard let rhs = to as? ArchiveAction else { return false }
-            return super.isEqual(to: to) &&
-                buildConfiguration == rhs.buildConfiguration &&
-                revealArchiveInOrganizer == rhs.revealArchiveInOrganizer &&
-                customArchiveName == rhs.customArchiveName
+        
+
+        static func == (lhs: ArchiveAction, rhs: ArchiveAction) -> Bool {
+            return
```

Comment:
Notice that we got rid of the `super.isEqual(to: to)` that delegates the comparison of some attributes to its parents.

Reply:
Yeah, polymorphic equality gets tricky when you move away from NSObject. I mostly tried to manually check the elements on the superclass as well which is not ideal; I think casting and using the superclass operator may work though.